reactive-banana/src/Reactive/Banana/Model.hs

```diff
@@ -2,6 +2,8 @@
     reactive-banana
 ------------------------------------------------------------------------------}
 {-# LANGUAGE RecursiveDo #-}
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
+{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
```

Comment:
I think the model really wants to be working with `data Stream a = a :> as`, but that's a change for another day. Rather than cluttering the model with exhaustive pattern matches that just use `error`, I've turned exhaustiveness checking off here.